monkey-runner/build.gradle

```diff
@@ -0,0 +1,24 @@
+task wrapper(type: Wrapper) {
+    gradleVersion = versions.gradleWrapperVersion
+}
+
+buildscript {
+    apply from: 'dependencies.gradle'
+
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        classpath gradlePlugins.gradle
+        classpath gradlePlugins.gradleAndroidCommandPlugin
+        classpath gradlePlugins.bintrayRelease
+    }
+}
+
+apply from: 'dependencies.gradle'
```

Comment:
I suppose this doesn't work because you're defining the buildscript repos and the dependencies for the root project, while you actually need them only in the subprojects. Try with:
```
apply from: 'dependencies.gradle'
subprojects {
    buildscript {
        repositories {
            jcenter()
        }
        dependencies {
            classpath gradlePlugins.gradle
            classpath gradlePlugins.gradleAndroidCommandPlugin
            classpath gradlePlugins.bintrayRelease
        }
    }
    repositories {
        jcenter()
    }
    version = '0.0.1'
}
```